Research/ResearchUI/iOS/Step View Controllers/RSDScrollingOverviewStepViewController.swift

```diff
@@ -126,6 +184,10 @@ open class RSDScrollingOverviewStepViewController: RSDOverviewStepViewController
         if placement == .body {
             
             scrollView.backgroundColor = background.color
+            
+            self.stepTitleLabel?.textColor = self.designSystem.colorRules.textColor(on: background, for: .heading1)
+            self.stepTitleLabel?.font = self.designSystem.fontRules.font(for: .heading1)
```

Comment:
Again, This is a change to every screen in every project that uses this. Just, no.

This has been set in the `RSDStepNavigationView` as `heading2`. This effects *every* screen in *every* app and should at the very least be called out as an override of the default behavior that is signed/dated.

Reply:
In the Figma PKU project, all step titles, whether they are in the body or header, are lato-bold 24. That font and size is described as heading1 (see the comment I added for you in Figma) in the font legend.  

Bold/24 does accurately translate to SageResearch's heading2 size in RSDFontRules.  However, labeling it as heading2 is confusing me, and doesn't reflect design's current understanding of that font type. Also, I did a global search for heading1 and I found that there is only one use of it in the entire repository, RSDStepTextFieldFeaturedCell.  

So, what are your thoughts on heading1 now being step titles and updating it to lato-bold/24?  Or is this a larger conversation we need to have with design about their understanding of where heading2 applies vs your coded implementation?